setup.py

```diff
@@ -69,46 +60,32 @@ def main():
         platforms=['unix', 'linux', 'osx', 'cygwin', 'win32'],
         author='Holger Krekel, Bruno Oliveira, Ronny Pfannschmidt, Floris Bruynooghe, Brianna Laugher, Florian Bruhin and others',
         author_email='holger at merlinux.eu',
-        entry_points=make_entry_points(),
+        entry_points={
+            'console_scripts': ['py.test = pytest:main']
+        },
         classifiers=classifiers,
         cmdclass={'test': PyTest},
-        # the following should be enabled for release
-        install_requires=install_requires,
-        extras_require=extras_require,
+        install_requires=['py>=1.4.29'],
+        extras_require={
+            ':python_version=="2.6" or python_version=="3.0" or python_version=="3.1"': ['argparse'],
+            ':sys_platform=="win32"': ['colorama'],
+        },
+        # pluggy is vendored in _pytest.vendored_packages
         packages=['_pytest', '_pytest.assertion', '_pytest._code', '_pytest.vendored_packages'],
         py_modules=['pytest'],
         zip_safe=False,
     )
 
 
-def cmdline_entrypoints(versioninfo, platform, basename):
```

Comment:
Hey @RonnyPfannschmidt, I wonder at which point where those alternative script names necessary, and why that's no longer the case?